### PR TITLE
rr: update to 5.7.0.

### PR DESCRIPTION
**Pull request by Nopey**
#### Testing the changes
- I tested the changes in this PR: **YES**

linux-6.patch is no longer needed, as it has been merged upstream.

an update-check script has been added for rr, it checks for their latest GitHub release

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by Nopey** [comment]
reset rr template revision to 1 as advised by the CI pipeline

File: srcpkgs/rr/update
Comment (classabbyamp) [inline srcpkgs/rr/update:1]: works fine without this

File: srcpkgs/rr/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT, BSD-2-Clause"
 homepage="http://rr-project.org/"
 distfiles="https://github.com/mozilla/rr/archive/${version}.tar.gz"
```
[the remark this answers is not in the document]
Comment (Nopey) [reply srcpkgs/rr/template]: Good catch, fixed.

File: srcpkgs/rr/update
Comment (Nopey) [reply srcpkgs/rr/update:1]: The `update` file is so `./xbps-src update-check rr` can detect when a new version of rr comes out.
No update scripts are strictly needed for building void packages

Comment (classabbyamp) [reply srcpkgs/rr/update:1]: I know, I'm saying it works without the update file. the file is for overrides in the case that update-check doesn't autodetect things properly

Comment (Nopey) [reply srcpkgs/rr/update:1]: Didn't know it did that, neat!
Removed `update`.

**Pull request merged**